### ISSUE TITLE
Images in gmail are broken due to Gmail image proxy

### ISSUE THREAD

**Issue by filviu** [matomo-org/matomo#6443]
Hi,

TL;DR
If you send email reports from a restricted piwik installation icons appear as broken images in gmail.

Longer description:

Gmail decided to start proxying images for users in order to break image based tracking (wether this is good or bad is not worth discussing here)

If for some reason your piwik installation is not public (for example I run one where only access_logs are used for data gathering and so is installed on an intranet) the graphs will load in the reports email (because are they sent as attachements) but flag icons, etc. will show as broken links because google is not able to load those in their proxy.

Whitelisting is only possible if you run google apps for domains. Could this be worked around (send those as attachments too?)

Cheers

**Comment by mattab** [comment]
Matomo email reports images should work fine in Gmail.
